DBConnection.cpp

```diff
@@ -13,87 +13,87 @@
 
 //Data Access Methods
 int DBConnection::prepareStatement(string sql)
-{	
-	prep_stmt = conn->prepareStatement(sql);
-	
-	return 0;
+{    
```

Comment:
The next thing we need to do is change the methods so that the opening curly bracket is on the same line as the method signature.